modules/oe_content_consultation/config/install/field.storage.node.oe_consultation_opening_date.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - datetime
+    - node
+id: node.oe_consultation_opening_date
+field_name: oe_consultation_opening_date
+entity_type: node
+type: datetime
+settings:
+  datetime_type: date
+module: datetime
+locked: false
+cardinality: 1
+translatable: true
```

Comment:
`translatable: false` as in Call for proposals. Dates usually are untranslatable.